elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
```

Comment:
I think this is a different context here. In binned spike train class the parameter `as_bool` was used to define the output. Here it is used internally, to clip a list of spiketrains to a binary representation but the output is not binary.  In this case the output is a sum/number of spiking neurons represented as a Time Histogram.
Maybe `clipping` (as a process) is more clear?  Or some other term?